config/initializers/okcomputer.rb

```diff
@@ -0,0 +1,21 @@
+# typed: strict
+# frozen_string_literal: true
+
+OkComputer.mount_at = 'status'
+OkComputer.check_in_parallel = true
+
+# Required
+OkComputer::Registry.register 'ruby_version', OkComputer::RubyVersionCheck.new
+
+# Optional
+OkComputer::Registry.register 'background_jobs', OkComputer::SidekiqLatencyCheck.new('default', 25)
```

Comment:
This check is listed as optional (comment above it) but not marked as optional below, it looks like.

Reply:
Fixed.